x-pack/plugins/security_solution/public/flyout/left/components/prevalence_details.tsx

```diff
@@ -63,16 +69,31 @@ export const PREVALENCE_TAB_ID = 'prevalence-details';
 const DEFAULT_FROM = 'now-30d';
 const DEFAULT_TO = 'now';
 
-const columns: Array<EuiBasicTableColumn<PrevalenceData>> = [
+interface PrevalenceDetailsRow extends PrevalenceData {
+  /**
+   * From datetime selected in the date picker to pass to timeline
+   */
+  from: string;
+  /**
+   * To datetime selected in the date picker to pass to timeline
+   */
+  to: string;
+}
+
+const columns: Array<EuiBasicTableColumn<PrevalenceDetailsRow>> = [
   {
     field: 'field',
     name: PREVALENCE_TABLE_FIELD_COLUMN_TITLE,
     'data-test-subj': PREVALENCE_DETAILS_TABLE_FIELD_CELL_TEST_ID,
+    render: (field: string) => <EuiText size="xs">{field}</EuiText>,
+    width: '55%',
   },
   {
     field: 'value',
     name: PREVALENCE_TABLE_VALUE_COLUMN_TITLE,
     'data-test-subj': PREVALENCE_DETAILS_TABLE_VALUE_CELL_TEST_ID,
+    render: (value: string) => <EuiText size="xs">{value}</EuiText>,
+    width: '55%',
```

Comment:
yeah I noticed that too. I don't think this PR introduced the bug, I think it's the previous one with the tooltip, but I'll fix it here if I find a good way

Reply:
I think this is actually a limitation of the EuiTable. As you can see in this video, for non-customized titles the table will cut the text and display `...`
https://github.com/elastic/kibana/assets/17276605/48ac8f98-1bb0-4570-82f7-d1af995b53b4

But when the title is customized (for example you wrap the text by an EuiTooltip, this is what happens
https://github.com/elastic/kibana/assets/17276605/0ffa09a9-2b3f-4a00-8dce-4ae9361e1c9c

Both videos are recorded off of the [EUI website](https://elastic.github.io/eui/#/tabular-content/tables#adding-sorting-to-a-table).

Not sure there is much to do here...
I had put the text on 2 rows instead of 1 to limit... but I'm not sure what to do next, other than repeating the value in the tooltip...